TLM/TLM/UI/MainMenu/VersionLabel.cs

```diff
@@ -8,10 +8,19 @@
 namespace TrafficManager.UI.MainMenu {
 	public class VersionLabel : UILabel {
 		public override void Start() {
-			size = new Vector2(MainMenuPanel.SIZE_PROFILES[0].MENU_WIDTH, MainMenuPanel.SIZE_PROFILES[0].TOP_BORDER); // TODO use current size profile
-			text = "TM:PE " + TrafficManagerMod.Version;
-			relativePosition = new Vector3(5f, 5f);
-			textAlignment = UIHorizontalAlignment.Left;
+
+            size = new Vector2(MainMenuPanel.SIZE_PROFILES[0].MENU_WIDTH, MainMenuPanel.SIZE_PROFILES[0].TOP_BORDER); // TODO use current size profile
+
+#if LABS
```

Comment:
If you merge `Version` with `Branch` you can remove these directives 😉
What do you think?

Reply:
I'd like to avoid using static stuff as much as possible as per #211

Reply:
I know but `static stuff` in context of complex objects but not just string (immutable ref type but passed by value)

Reply:
I've left this as-is for now.